README.md

```diff
@@ -76,32 +100,83 @@ For instance, `wnfs/APPEND` is an action for WebNative filesystem paths. The act
+  { "with": "wnfs://alice.example.com/pictures/vacation/hawaii/", "can": "wnfs/OVERWRITE"}
+];
 
-As a practical matter, since scopes form a group, you can be fairly loose: order doesn’t matter, and merging resources can be quite broad since the more capable of any overlap will take precedence (i.e. you don’t need a clean separation).
+merge(ScopeA, ScopeA) == [
+   {"with": "wnfs://alice.example.com/pictures/", "can": "wnfs/APPEND"},
+   {"with": "wnfs://alice.example.com/pictures/vacation/", "can": "wnfs/OVERWRITE"}
+   // Note that ("/pictures/vacation/" x APPEND) has become redundant, being contained in ("/pictures/" x APPEND)
+];
+```
 
 ## 2.5 Delegation
 
-Delegation is the act of granting another principal (the delegate) the capability to use a resource that another has (the delegator). This MUST be proven by a "witness", which is either the signature of the owning principal, or a UCAN that has access to that capability in its scope.
+Delegation is the act of granting another principal (the delegate) the capability to use a resource that another has (the delegator). This MUST be proven by a "witness", which is either the signature of the owning principal, or a UCAN that has access to that capability in its capability scope.
 
-Each direct delegation leaves the scope of the action at the same level, or diminishes it. The only exception is in "rights amplification", where a delegation MAY be proven by one-or-more witnesses of different types, if part of the resource's semantics. 
+Each direct delegation leaves of the action at the same level, or diminishes it. The only exception is in "rights amplification", where a delegation MAY be proven by one-or-more witnesses of different types, if part of the resource's semantics. 
+
+Note that delegation is a seperate concept from invocation [§2.8](#28-invocation). Delegation is the act of granting a capability to another, not the act of using it (invocation) which has additional requirements.
 
 ## 2.6 Attenuation
 
-The process of diminishing rights in a delegation chain.
+The process of constraining the capabilities in a delegation chain.
 
 ## 2.7 Revocation
 
 Revocation is the act of invalidating a UCAN after the fact, outside of the limitations placed on it by the UCAN's own fields (such as its expiry). 
 
 In the case of UCAN, this MUST be done by a witness issuer DID. For more on the exact mechanism, see the revocation validation section.
 
+## 2.8 Invocation
+
+UCANs are used to delegate capabailities between DID-holding agents, eventually terminating in an "invocation" of those capabilities. This is when the capability is exercised to perform some task on the resource. Note that **the only agent that is allowed to perform an action with a UCAN MUST be the one holding the DID private key associated with the `aud` field**. For more on the specifics of this validation, see [§5.2.1](#521-invocation-recipient-validation).
```

Comment:
The "This" opening the second sentence could be made more concrete by repeating invocation. Something like:

> Invocation occurs when the capability is exercised to perform some task on the resource.

Also, a small nit. "the resource" in the sentence above could be replaced with "a resource". We aren't referring to a specific resource, and "a" is better at implying some resource, doesn't matter which, just that it is associated with the capability.